doc/advanced-puppet-master.md

```diff
@@ -8,7 +8,11 @@ Please note the following caveats:
 
 0. You will need to deploy your Puppet code to an environment on your Puppet Master prior to running `octocatalog-diff` for that environment. `octocatalog-diff` does not deploy code for you.
 
-0. You will need to configure authorization for one or more whitelisted certificates on your Puppet Master. The default permissions allow a node to retrieve its own catalog via the API, but you need a certificate for `octocatalog-diff` that permits it to retrieve any catalog. See the [Certificate authorization](#certificate-authorization) section below.
+0. You will need to configure authorization for one or more whitelisted certificates on your Puppet Master. The default permissions allow a node to retrieve its own catalog via the API, but you need a certificate for `octocatalog-diff` that permits it to retrieve any catalog. See the [Certificate authorization](#certificate-authorization) section below. If you are using Puppet Enterprise and use
+the Puppet Master v4 API you may also use a Puppet Enterprise RBAC token. The user the token was
+issued to will need the "Puppet Server Compile catalogs for remote nodes" permission.
```

Comment:
Is this a user-owned token, or does the token live on its own?

If it's a user-owned token, I'd say `The user account which owns the token will need the .... permission`.
If tokens are not owned by users, I'd say `The token will need the .... permission`

Reply:
It is user-owned, I'll update the docs. Great feedback, thanks!